### ISSUE TITLE
.NET SDK is not strong named.

### ISSUE THREAD

**Issue by danielwspie** [Keeper-Security/secrets-manager#404]
We have a library of .NET Framework 4.7.2 assemblies that require referenced assemblies be strong named, thus including the NuGet package Keeper.SecretsManager Version 16.4.0 results in the following error:

`System.IO.FileLoadException: 'Could not load file or assembly 'SecretsManager, Version=16.4.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. `

Based on this Microsoft documentation from Sept 2021, it recommends `You should strong name your open-source .NET libraries. Strong naming an assembly ensures the most people can use it, and strict assembly loading only affects .NET Framework.`. 
https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/strong-naming

Please consider strong naming your .NET SDK assemblies.

**Comment by DevOma75** [comment]
I have signed the file in my build process but also would like to get a strong named file from Nuget package without manual steps

**Comment by danielwspie** [comment]
@maksimu yes, due to third party dependencies, we will be on this .NET Framework 4.7.2 infrastructure for the foreseeable future, so the inclusion of this will make our interaction with the Keeper library a much easier experience. Also note, this is the only NuGet package we've encountered that isn't strongly named.

**Comment by DevOma75** [comment]
One library is enough - as it is possible to reference signed assemblies from unsigned projects - the change will not affect anyone who used it in that way. But for all who are working with signed libraries it will be more comfortable - I have to use ildasm and ilasm in my build process to sign the assembly before referencing. That step would not be needed after you sign the assembly in NuGet package - thanks

**Comment by danielwspie** [comment]
@maksimu  any update on this issue? Looking at https://github.com/Keeper-Security/secrets-manager/tree/master/sdk/dotNet it looks like it was attempted to include this in 16.6.6, but it looks like https://github.com/Keeper-Security/secrets-manager/actions/workflows/publish.nuget.strong.named.yml failed
